background/services/name/index.ts

```diff
@@ -205,6 +210,10 @@ export default class NameService extends BaseService<Events> {
     const { address: normalizedAddress, network } =
       normalizeAddressOnNetwork(addressOnNetwork)
 
+    if (!this.cachedResolvedNames[network.family][network.chainID]) {
+      this.cachedResolvedNames[network.family][network.chainID] = {}
+    }
```

Comment:
```suggestion
    this.cachedResolvedNames[network.family][network.chainID] ??= {}
```

Personal idiomatic preference.

Reply:
Though if we do this, why do the `fromEntries` dance above? Feels unnecessary, and I think we should keep this one rather than that one.